webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -1570,6 +1590,9 @@ public static ByteArrayOutputStream requestsToZip(ServletContext context, String
         jsonObj.put("version", Utils.getProjectName() + "-" + Utils.getVersion());
         jsonObj.put("generationDate", generationDate);
         jsonObj.put("generationUrl", generationUrl);
+        if (wcifHelper != null) {
+            jsonObj.put("schedule", wcifHelper.getSchedule());
```

Comment:
Out of curiosity, why are we putting the schedule in here? I think this JSON file is only read by the workbook assistant, and we don't have any plans to have it start reading the schedule.

That said, I don't think there's any downside to putting the schedule here, and it might be nice to have a snapshot of the schedule we used for generating scrambles, so I'm ok with leaving this here. Just want to understand why it's here.

Reply:
It's mostly for the automated test I'm about to write. I also wanna check if the PDFs are in the correct order. It looks like we can change schedule after a competition is over, so using the online schedule might not be the best option.